app/database/models.py

```diff
@@ -14,7 +14,7 @@ class User(Base):
     full_name = Column(String)
     description = Column(String, default="Happy new user!")
     avatar = Column(String, default="profile.png")
-
+    language = Column(Integer, ForeignKey("languages.id"))
```

Comment:
Move down near the other foreign keys (near owner_id)

Reply:
Move done where? There are no other foreign keys there. Do you mean after the is_active?

Reply:
You are right, the preview confused me.
Anyhow, I think it's good idea to place it after is active (and all other fields)

Reply:
Sure, I'll change it in the next push :)